pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>3.9</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.222.4</jenkins.version>
+        <jenkins.version>2.300-rc31329.23a941f5b44f</jenkins.version> <!-- TODO https://github.com/jenkinsci/jenkins/pull/5599 -->
```

Comment:
I recommend marking downstream PRs like this as draft, to make sure they are not accidentally merged prematurely.